vimeo-networking/src/main/java/com/vimeo/networking/callers/GetRequestCaller.java

```diff
@@ -86,6 +86,27 @@
                 }
             };
 
+    /**
+     * Used in association with
+     * {@link VimeoClient#getContent(String, CacheControl, Caller, String, Map, String, VimeoCallback)} or
+     * {@link VimeoClient#getContentSync(String, CacheControl, String, Map, String, Caller)}
+     * to get an {@link com.vimeo.networking2.AppConfiguration} response from an API endpoint. This caller will call
+     * through to
+     * {@link VimeoService#getAppConfigurationMoshi(String, String, Map, String)} (String, String, Map, String)}
+     */
+    public static final Caller<com.vimeo.networking2.AppConfiguration> APP_CONFIGURATION_MOSHI =
```

Comment:
I had created the class `MoshiGetRequestCaller` to hold all Moshi related callers. Should this be moved into that class? Does it cause any complications in the integration?